common/db.go

```diff
@@ -169,14 +171,16 @@ func NewDBJob(id *big.Int, streamID string,
 
 func DBJobToEthJob(j *DBJob) *lpTypes.Job {
 	return &lpTypes.Job{
-		JobId:              big.NewInt(j.ID),
-		StreamId:           j.streamID,
-		MaxPricePerSegment: big.NewInt(j.price),
-		Profiles:           j.profiles,
-		BroadcasterAddress: j.broadcaster,
-		TranscoderAddress:  j.Transcoder,
-		CreationBlock:      big.NewInt(j.startBlock),
-		EndBlock:           big.NewInt(j.endBlock),
+		JobId:               big.NewInt(j.ID),
+		StreamId:            j.streamID,
+		MaxPricePerSegment:  big.NewInt(j.price),
+		Profiles:            j.profiles,
+		BroadcasterAddress:  j.broadcaster,
+		TranscoderAddress:   j.Transcoder,
+		CreationBlock:       big.NewInt(j.startBlock),
+		EndBlock:            big.NewInt(j.endBlock),
+		TotalClaims:         big.NewInt(j.Claims),
+		FirstClaimSubmitted: j.Claims > 0,
```

Comment:
If we actually proceed with a claim in that situation, the claimmanager would abort it anyway because the ClaimID returned by `db.InsertClaim` doesn't take into account previous failed attempts.

This is definitely something that should be made more robust, but it's a bit more involved since it means fixing up the claim IDs in receipts associated with the failed claims. Or maybe the flow needs to be refactored in some other way. Think we can just make a ticket to track the issue for now?

https://github.com/livepeer/go-livepeer/blob/b26253aa6996b2069722ec658222f230ee81d847/eth/claimmanager.go#L366-L382

Reply:
Modifying the `db.countClaims` query to exclude statuses of "FAIL submit tx" and "FAIL check tx" might help with this (it is used in both `db.GetJob` and `db.InsertClaim`), but I need to look at things a little more closely.

Reply:
Ah right. Yeah I agree that we should file a ticket and come back to this problem separately as it is a bit more involved than I originally thought